sklearn/dummy.py

```diff
@@ -120,7 +120,8 @@ def fit(self, X, y, sample_weight=None):
         if not self.sparse_output_:
             y = np.atleast_1d(y)
 
-        self.output_2d_ = y.ndim == 2
+        self.output_2d_ = (y.ndim == 2) and (y.shape[1] > 1)
```

Comment:
So you're not using `column_or_1d` because of the reshape below right?

Reply:
yes